src/main.rs

```diff
@@ -42,7 +42,7 @@ Options:
 ";
 
 #[derive(Debug, RustcDecodable)]
-struct MainArgs {
+pub struct MainArgs {
```

Comment:
You don't need this to be `pub` I think, `check` is a submodule.
```suggestion
struct MainArgs {
```